src/block/tile/BrewingStand.php

```diff
@@ -108,4 +126,125 @@ public function getInventory(){
 	public function getRealInventory(){
 		return $this->inventory;
 	}
+
+	private function checkFuel(Item $item) : void{
+		$ev = new BrewingFuelUseEvent($this);
+		$ev->call();
+
+		if($ev->isCancelled()) {
+			return;
+		}
+
+		$item->pop();
+		$this->inventory->setFuel($item);
+
+		$this->maxFuelTime = $this->remainingFuelTime = $ev->getFuelTime();
+	}
+
+	private function canBrew() : bool{
+		if($this->inventory->getIngredient()->isNull()){
+			return false;
+		}
+
+		for($i = 1; $i <= 3; ++$i){
+			$input = $this->inventory->getItem($i);
+			if($input->isNull()) {
+				continue;
+			}
+
+			if($this->position->getWorld()->getServer()->getCraftingManager()->matchBrewingRecipe($input, $this->inventory->getIngredient()) !== null){
+				return true;
+			}
+		}
+
+		return false;
+	}
+
+	public function onUpdate() : bool{
+		if($this->closed){
+			return false;
+		}
+
+		$this->timings->startTiming();
+
+		$prevBrewTime = $this->brewTime;
+		$prevRemainingFuelTime = $this->remainingFuelTime;
+		$prevMaxFuelTime = $this->maxFuelTime;
+
+		$ret = false;
+
+		$fuel = $this->inventory->getFuel();
+		$ingredient = $this->inventory->getIngredient();
+
+		$canBrew = $this->canBrew();
+
+		if($this->remainingFuelTime <= 0 and $canBrew and $fuel->equals(VanillaItems::BLAZE_POWDER(), true, false)) {
+			$this->checkFuel($fuel);
+		}
+
+		if($this->remainingFuelTime > 0){
+			if($canBrew){
+				if($this->brewTime == 0){
+					$this->brewTime = self::BREW_TIME;
+					--$this->remainingFuelTime;
+				}
+
+				--$this->brewTime;
+
+				if($this->brewTime <= 0){
+					for($i = 1; $i <= 3; ++$i){
```

Comment:
This shouldn't be hardcoded. Please update this to use the constants added on the master branch.